content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,212 @@
+| enableInsecureLogin | RBAC ensures secure login. | `false` |
+| serviceType | The service type for the dashboard. Supported values are ClusterIP, LoadBalancer, and NodePort. | `ClusterIP` |
+| skipLogin | Enables or disables the skip login option on the dashboard. | `false` |
+| enableInsecureLogin | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| ingress:enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
+
+## Usage
+
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. This integrated pack supports the following clouds:
+
+- Amazon Web Services (AWS)
+- Amazon Elastic Kubernetes Service (EKS)
+- Azure
+- Canonical Metal as a Service (MAAS) 
+- Google Cloud Platform (GCP)
+- OpenStack
+- VMware vSphere
+
+
+The Spectro Kubernetes Dashboard pack has default settings for **Access** and **Identity Provider**. These settings automatically install the latest version of the Spectro Proxy pack when you create the cluster profile.
+
+Changing these default settings may require some additional configuration.
+<br /> 
+
+#### Access
+
+The default setting is **Private**. If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
+
+However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](/integrations/frp) guide.
+
+<br /> 
+
+#### Identity Provider 
+
+The default setting is **Palette**.
+
+<br />
+
+- **Palette**: No configuration is needed. This setting makes Palette the IDP, so any user with a Palette account in the tenant and the proper permissions to view and access the project's resources is able to log into the Kubernetes dashboard.
+
+- **Inherit from Organization**: This setting requires you to configure OpenID Connect (OIDC) in Tenant Settings. In Tenant Admin scope, navigate to **Tenant Settings > SSO**, choose **OIDC**, and provide your third-party IDP details. For more information, check out the [SSO Setup](/user-management/saml-sso) guide. 
+
+- **None**: This setting requires you to configure OIDC manually in the Kubernetes pack.
+
+### Manually Configure OIDC
+
+You only need to configure OIDC manually if you change the **Identify Provider** setting to **None**. The basic method to enable OIDC can be used for all cloud services except Amazon EKS. 
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Basic OIDC Setup" key="Other Clouds">
+
+
+To configure OIDC manually for clusters managed by most cloud providers, follow these steps: 
+
+<br />
+
+1. Copy ``oidc-`` configuration lines in the example and add them to the Kubernetes pack under the ``extraArgs`` parameter section. Enter your third-party provider details in quotes. <br /><br />
```

Comment:
```suggestion
1. Copy ``oidc-`` configuration lines in the following code snippet and add them to the Kubernetes pack under the ``extraArgs`` parameter section. Enter your third-party provider details in quotes. <br /><br />
```
I got confused when reading the example and thought I missed something 😵‍💫 . This could just be me, but I think specifying the code snippet will reduce the chances this happening to someone else.